src/hotspot/share/gc/g1/heapRegionSet.hpp

```diff
@@ -227,6 +227,7 @@ class FreeRegionList : public HeapRegionSetBase {
   // are taken care of separately, to allow a rebuild.
   void abandon();
 
+  void verify_region_to_remove(HeapRegion* curr, HeapRegion* next) NOT_DEBUG_RETURN;
```

Comment:
Please move this to the private section, right below the `add_list_common`-functions would be fine.